lambda_src/drivers/process_https.py

```diff
@@ -90,21 +92,31 @@ def process_row(
             req_headers['authorization'] = req_auth['authorization']
         elif 'headers' in req_auth:
             req_headers.update(req_auth['headers'])
-
+        elif 'body' in req_auth:
+            json_body = (
+                loads(req_auth['body'])
+                if isinstance(req_auth['body'], str)
+                else req_auth['body']
+            )    
+            if json:
+                raise ValueError(f"both auth body and json present")
+            else:
+                json = json_body
```

Comment:
on second read, we only use this variable once, so it's probably best to in-line the above code here